dev-util/systemtap/systemtap-3.3.ebuild

```diff
@@ -0,0 +1,81 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit linux-info autotools eutils python-single-r1
+
+DESCRIPTION="A linux trace/probe tool"
+HOMEPAGE="https://www.sourceware.org/systemtap/"
+SRC_URI="https://www.sourceware.org/${PN}/ftp/releases/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="alpha amd64 arm ~arm64 ia64 ~mips ppc ppc64 s390 ~sparc x86"
+IUSE="libvirt sqlite"
+
+RDEPEND=">=dev-libs/elfutils-0.142
+	sys-libs/libcap
+	${PYTHON_DEPS}
+	libvirt? ( >=app-emulation/libvirt-1.0.2 )
+	sqlite? ( dev-db/sqlite:3 )"
+DEPEND="${RDEPEND}
```

Comment:
possibly missing setuptools?

Reply:
Why would it be necessary?

Reply:
because it is used http://sourceware.org/git/gitweb.cgi?p=systemtap.git;a=blob;f=python/setup.py;h=aea5ce9c8b1aef53dfc220fd9e33c6c7951e97c0;hb=48867d1cface9445d58e3c6e14497770b7eb77e1

Reply:
Ok. I'll add it. Is there a bug report on the tracker we can tie this to?